README.md

```diff
@@ -189,6 +189,7 @@ Running the game over internet currently requires setting up a relay.
 * Relay Join - StartClientLobby() in [Assets/Scripts/ConnectionManagement/ConnectionState/OfflineState.cs ](Assets/Scripts/ConnectionManagement/ConnectionState/OfflineState.cs)
 * Relay Create - StartHostLobby() in [Assets/Scripts/ConnectionManagement/ConnectionState/OfflineState.cs ](Assets/Scripts/ConnectionManagement/ConnectionState/OfflineState.cs)
 * Authentication - EnsurePlayerIsAuthorized() in [Assets/Scripts/UnityServices/Auth/AuthenticationServiceFacade.cs ](Assets/Scripts/UnityServices/Auth/AuthenticationServiceFacade.cs)
+* Authentication - Profile management for ParrelSync/local instances - GetProfile() in [Assets/Scripts/Utils/ProfileManager.cs](Assets/Scripts/Utils/ProfileManager.cs)
```

Comment:
> Wouldn't it be better to add a permalink to the line where the method is implemented?

@RikuTheFuffs yeah, thought about it. From a future proofing perspective, there's a higher chance a line will change than a filename. If there was a way to link to method names instead of lines, that would have been awesome. But I think lines are too flaky to link to directly.
I could add a comment in code for future folks updating this, telling them to update the line, but I think this is simpler.

Reply:
@SamuelBellomo I think github's permalink don't care about line numbers, which is why I suggested them. Line wouldn't be future-proof

Reply:
Any docs on this? Cause the link generated does contain a line number in the URL, not a method name or line to track. This with a branch name only, not sure how opening such link could have enough context to track the actual line of code from link creation to the new state it's currently in.